scripts/nemoclaw-start.sh

```
@@ -79,6 +82,52 @@ os.chmod(path, 0o600)
+  # entry is gone too, so skip gracefully.
+  if [ ! -w "$config_path" ]; then
+    echo "[security] gateway config already locked: $config_path"
+    return
+  fi
+
+  # In the standard path, wait for the gateway to finish its initial config
+  # write (token generation) before locking.  Skip in the exec path where
+  # no gateway is running — avoids a 30s hang.
+  if [ "$wait_for_token" = "true" ]; then
+    local i
+    for i in $(seq 1 30); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)